### ISSUE TITLE
Errors in Django tests aren't being reported

### ISSUE THREAD

**Issue by christophertubbs** [NOAA-OWP/DMOD#494]
Got a message like the following in a failing test:

```
 python/services/evaluationservice/dmod/evaluationservice
======================================================================
/home/runner/work/DMOD/DMOD/python/services/evaluationservice/dmod/evaluationservice
No line stating the number of tests run could be found.
```

That's _supposed_ to state info on what failed. Nothing was found when parsing the output. This needs to be fixed.

**Comment by aaraney** [comment]
Yeah, I stumbled across this in fixing #492. I think I know what is going on. Ill get a PR in here soon.

**Comment by aaraney** [comment]
They will be reported if you throw the verbose flag with `./scripts/run_tests.sh`. Otherwise the default behavior of `./scripts/test_django.py` hides the output.

**Comment by christophertubbs** [comment]
I _may_ have fixed it with a PR yesterday. Had something to do with trying to parse the wrong string to find the number of tests, I _believe_.

Best way to test is to throw a something like `self.fail("test")` at the top of one of the unit tests and see what happens when you run `test_django.py`. The output should look as close as possible to the other results in `run_tests.sh`. It's gonna look strange, but that's the nature of the beast when you using fairly different tools.
